canonical/canonical.com
# Fix homepage tabs on small screens

## Done

- Fixed tabs not maintaining row width
- Bumped Vanilla version to latest
- Removed unnecessary custom styles

## QA

- View the site locally in your web browser at: http://0.0.0.0:8002/
- See that the fix has been made

## Issue / Card

Fixes # https://warthogs.atlassian.net/browse/WD-5178

## Screenshots

Before: 
![Screenshot from 2023-08-01 10-56-56](https://github.com/canonical/canonical.com/assets/17607612/6d024d6e-732a-406f-b7dc-6e3f13fa5a5a)

After: 
![image](https://github.com/canonical/canonical.com/assets/17607612/058144e4-0290-45a3-b1bb-67c920f02aab)
